README.md

```diff
@@ -45,15 +45,51 @@ tomorrowAsString(new Date('2015-01-01')); // '2015-01-02';
 
 ## API
 
+### of
+`String | [Number] | Void -> Date`
```

Comment:
If people want the current date they can use `Date.now()`. I don't think that should be supported here.